fullscreen.bs

```diff
@@ -155,8 +155,18 @@ steps:
 <h2 id=api>API</h2>
 
 <pre class=idl>
+enum <dfn>FullscreenNavigationUI</dfn> {
+  "auto",
+  "show",
+  "hide"
+};
+
+dictionary <dfn>FullscreenOptions</dfn> {
+  FullscreenNavigationUI navigationUI = "auto";
+};
```

Comment:
I'd prefer placing these at the end of the IDL block. It's more common to start with the concrete things than the primitives, I think.

Reply:
I believe there's quite a mix on that front. I checked 5 cases at "random" before getting bored. Dictionaries first:
* https://drafts.csswg.org/cssom-view/#extensions-to-the-window-interface
* https://encoding.spec.whatwg.org/#interface-textdecoder
Dictionaries last:
* https://dom.spec.whatwg.org/#interface-eventtarget
* https://wicg.github.io/background-fetch/#background-fetch-manager
* https://w3c.github.io/sensors/#the-sensor-interface

I'll go with Dave's suggested order here since it reads fine to me.